portal/src/graphql/portal/BotProtectionConfigurationScreen.tsx

```diff
@@ -181,7 +280,8 @@ function ProviderCard(props: ProviderCardProps) {
   } = useSystemConfig();
 
   return (
-    <div
+    <button
+      type="button"
```

Comment:
Maybe not related to this pr, but I think we can replace these inline styles with css classes.
Define a class which only applies to the element when the button is in disabled state, so we don't need to use the `style` attribute.